packages/core-utils/src/route.js

```diff
@@ -0,0 +1,264 @@
+/** Looks up an operator from the provided configuration */
+export function getTransitOperatorFromId(id, transitOperators) {
```

Comment:
So, I think this function needs the jsdoc to be clarified a bit. I just found a bug in SEPTA where the agencyId is being passed to this function instead of the split route ID.